scoring.js

```diff
@@ -0,0 +1,79 @@
+const calculateScore = (player) => {
+    switch (player.position) {
+        case 'QB':
+            return calculateTotalPassing(player) + calculateTotalRushing(player)       
+
+        case 'RB': 
+            return   calculateTotalRushing(player) + calculateTotalReceiving(player) + calculateTotalKickReturn(player) + calculateTotalPuntReturn(player)
+
+        case 'WR': 
+            return   calculateTotalRushing(player) + calculateTotalReceiving(player) + calculateTotalKickReturn(player) + calculateTotalPuntReturn(player)
+
+        case 'TE': 
+            return    calculateTotalReceiving(player)  
```

Comment:
Watch your spacing. It's minor but extra spaces and newlines tells people the code was written by someone junior. Make sure you have `Format on Save` turned on for VS Code. It will solve a lot of this for you.